### ISSUE TITLE
求作者能提供一份推理脚本

### ISSUE THREAD

**Issue by whiteyyc** [LetterLiGo/SafeEar#41]
class Safeear:
    def __init__(self,config_path: str = "model_inference/audio/SafeEar/config/train19.yaml" ):
        self.cfg = OmegaConf.load(config_path)

        """
        初始化推理系统，加载模型和配置。
        """
        # 加载检测模型 (detect_model)
        self.detect_model = hydra.utils.instantiate(self.cfg.detect_model).to(device)
        checkpoint = torch.load(self.cfg.detect_model.location, map_location=device)
        self.detect_model.load_state_dict(self._adjust_state_dict(checkpoint['state_dict']))
        self.detect_model.eval()

    def _adjust_state_dict(self, state_dict):
        """
        调整模型权重的键名，去除 "detect_model." 前缀。
        Args:
            state_dict (dict): 原始权重字典。
        Returns:
            dict: 调整后的权重字典。
        """
        return {k.replace("detect_model.", ""): v for k, v in state_dict.items() if "detect_model" in k}

    def inference(self, audio_path: str) -> Dict:

        acmodel = Acoustic_features(self.cfg)
        acoustic_tokens = acmodel.decouple_acoustic_feats(audio_path)

        raw_logits, raw_feature = self.detect_model(acoustic_tokens)
        probs = torch.softmax(raw_logits, dim=-1)
        prob_spoof = probs[:, 1].item()  # 类别1为spoof
        score = round(prob_spoof * 100, 2)  # 保留两位小数

        label = "虚假" if prob_spoof >= 0.5 else 0  # 1=spoof, 0=bonafide

        return label,prob_spoof

这是我写的推理代码，希望作者能否看一下这个实现逻辑是否正确。

**Comment by wszz123** [comment]
xd，推理脚本你能正确运行了吗，求分享

**Comment by waerf** [comment]
xd，推理脚本你能正确运行了吗，求分享

**Comment by Wangtk311** [comment]
您好，请问您能看看我实现的这份推理方案是否正确吗？谢谢您！
https://github.com/Wangtk311/SafeEar-Inference-Test-Script
